src/flow/ICFlowSummary.php

```diff
@@ -84,7 +84,7 @@ protected function drawView() {
           pht(
             'Issue `%s` in order to create a new local tracking branch '.
             'starting from your current branch (%s).',
-            'flow branch branchname',
+            'arc flow branchname [upstream_branch]',
```

Comment:
Use `<branchname>` for mandatory arguments. Or make `[branchname]` for optional.